Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'figaro'
 gem 'jbuilder', '~> 2.0'
 gem 'pg'
 gem 'puma'
-gem 'rails', '5.0.0.1'
+gem 'rails', '5.0.1'
```

Comment:
Generally, don't update `rails` lightly. Dedicate a commit or even a branch to it. When doing so, review why are you doing it, and also make sure to follow the rails upgrade guides (http://guides.rubyonrails.org/upgrading_ruby_on_rails.html)

This time is no big deal, next one remember it